app.js

```diff
@@ -30,36 +28,51 @@ global.ADMIN_COMMANDS = require('./commands/admin.js');
 global.MOD_COMMANDS = require('./commands/mod.js');
 global.welcomeImage = "https://s26.postimg.cc/8x8hnunux/Reddit_Link.png";
 
+var winston = require('winston');
```

Comment:
This dependency should be at the top of the file I think. 

While we are cleaning up: 
1. Can we move GAME_ROLES to their own file? 
2. Can we move all the local file dependency to all be together under the node dependencies.
3. Can we add a global.USER_LEVEL/global.MEMEBER_LEVEL = 0 (IDK which we decided on in terms of language.)